examples/mqtt-client/mqtt-client.c

```diff
@@ -351,7 +351,17 @@ mqtt_event(struct mqtt_connection *m, mqtt_event_t event, void *data)
     break;
   }
   case MQTT_EVENT_SUBACK: {
+    #if MQTT_311
+    mqtt_suback_event_t *suback_event = (mqtt_suback_event_t*) data;
```

Comment:
code style: space before the *, no space after the closing paren. Try uncrustify scripts in `tools/`